### ISSUE TITLE
az apic service returns 400 Bad Request when importing APIs from APIM

### ISSUE THREAD

**Issue by aw230012** [Azure/api-center-preview#36]
# Description

When trying to import APIs from an APIM instance into APIC using Azure CLI's ```az apic import-from-apim``` command in PowerShell, the result is always a 4000 BAD REQUEST with an error ```Object reference not set to an instance of an object```. I just installed the extension today to work with apic, so it should be a newer version.

# Reproduction steps

1. Retrieve the APIM ID
2. Concatenate "/apis/*" to the end of the APIM ID into a new var $apiIDs
3. Execute the command ```az apic service import-from-apim --service-name <service name> --resource-group <rg-name> --source-resource-ids $apiIDs```

# Expected behavior

The APIs should import.

# Actual behavior

Always receive a 400 BAD REQUEST with an "Object reference not set to an instance of an object" error.

# Screenshots

# Additional context

Just installed the APIC extension today, so the version is newer. Followed the examples in the APIC import from APIM Microsoft Documentation.

**Comment by juliajuju93** [comment]
@aw230012, have you configured your managed identity correctly? Could you please clarify if you followed all the steps here?

https://learn.microsoft.com/en-us/azure/api-center/import-api-management-apis?tabs=portal#add-a-managed-identity-in-your-api-center

**Comment by aw230012** [comment]
> @aw230012, have you configured your managed identity correctly? Could you please clarify if you followed all the steps here?
> 
> https://learn.microsoft.com/en-us/azure/api-center/import-api-management-apis?tabs=portal#add-a-managed-identity-in-your-api-center

Yes I did verify the managed identity is correct. I am able to import the APIs one at a time. I only receive the 400 BAD REQUEST when trying to do the wildcard import as shown in the documentation.

**Comment by juliajuju93** [comment]
@aw230012 :

We recently released a new CLI extension version with updated functionalities, resolving some of the bugs you mentioned. Could you please run this again and let us know if it worked this time.
